backend/api/auth/BUILD

```diff
@@ -1,8 +1,12 @@
 app_deps = [
+    "//:jackson",
     "//:lombok",
+    "//:springboot",
```

Comment:
I believe all the apps have jackson lombok springboot and log  so that would probably be our starting point